fiddler/manifest.json

```diff
@@ -0,0 +1,81 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Fiddler provides visibility and explainability into AI model performance",
+    "title": "Fiddler",
+    "media": [
+      {
+        "caption": "Fiddler Model Performance Management",
+        "image_url": "images/fiddler-datadog.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Fiddler Model Performance Management",
```

Comment:
Please add different captions for each image

Reply:
Changed captions